app/lib/authz-middleware.js

```diff
@@ -7,11 +7,18 @@ const userRoles = {
   admin: 'admin',
   editor: 'editor',
   visitor: 'visitor',
+  teamLead: 'teamlead',
```

Comment:
The value here should match the related front end changes currently being worked on -> `teamLead: 'team_lead'`